config/routes.php

```diff
@@ -166,6 +166,11 @@
         ['controller' => 'Modules', 'action' => 'relationshipsJson'],
         ['pass' => ['id', 'relation'], '_name' => 'modules:relationshipsJson']
     );
+    $routes->connect(
+        '/:object_type/view/:id/resourcesJson/:relation',
```

Comment:
we need to adjust this URL since neither `object_type` nor  object `id` are needed in this context